build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -3,8 +3,16 @@
   <Import Project="compile/Microsoft.DotNet.Cli.LzmaArchive.targets" />
   <Import Project="crossgen/Microsoft.DotNet.Cli.Crossgen.targets" />
 
+  <PropertyGroup>
+    <BinaryToCorehostifyRelDir>runtimes/any/native</BinaryToCorehostifyRelDir>
+    <CoreSDKDir>$(RepoRoot)/resources/core-sdk</CoreSDKDir>
```

Comment:
not sure I am crazy about 'core-sdk' as a name for this directory.

How about `$(RepoRoot)/resources/MSBuild`  or `$(RepoRoot)/resources/MSBuildImports`.

Note that our .props and .targets are getting deleted from our repo now that they live in the dotnet/sdk repo.